pyanaconda/bootloader.py

```diff
@@ -1010,6 +1009,9 @@ class GRUB(BootLoader):
 
     _serial_consoles = ["ttyS"]
 
+    def _config_dir(self):
```

Comment:
@bwann I was matching the updated API from the grub2 version, but I missed a thing. It needs `@property` decorator. Sorry about that.